init-metadatastore.sh

```diff
@@ -4,6 +4,7 @@ export MDS_DATABASE=mds-test
 export MDS_TIMEZONE=US/Eastern
 export MDS_MONGO_USER=tom
 export MDS_MONGO_PWD=jerry
+export MDS_PORT=27017
```

Comment:
Historical note: early versions of MDS failed to parse this correctly as an integer, which is why it was missing here. But now it works fine.